test/e2e/atlas/kubernetes_operator_install_test.go

```diff
@@ -0,0 +1,404 @@
+		projectName := "MyK8sProject"
+
+		cmd := exec.Command(cliPath,
+			"kubernetes",
+			"operator",
+			"install",
+			"--targetNamespace", operatorNamespace,
+			"--projectName", projectName,
+			"--import",
+			"--kubeContext", context)
+		cmd.Env = os.Environ()
+		resp, inErr := cmd.CombinedOutput()
+		req.NoError(inErr, string(resp))
+		a.Equal("Atlas Kubernetes Operator installed successfully\n", string(resp))
+
+		checkDeployment(t, operator, operatorNamespace)
+
+		projectSecret := &corev1.Secret{}
+		inErr = operator.getK8sObject(
+			client.ObjectKey{Name: fmt.Sprintf("mongodb-atlas-%s-api-key", prepareK8sName2(projectName)), Namespace: operatorNamespace},
+			projectSecret,
+			false,
+		)
+		req.NoError(inErr)
+
+		orgSecret := &corev1.Secret{}
+		inErr = operator.getK8sObject(
+			client.ObjectKey{Name: "mongodb-atlas-operator-api-key", Namespace: operatorNamespace},
+			orgSecret,
+			false,
+		)
+		req.Error(inErr)
+
+		akoProject := &akov1.AtlasProject{}
+		err = operator.getK8sObject(
+			client.ObjectKey{Name: prepareK8sName2(projectName), Namespace: operatorNamespace},
+			akoProject,
+			true,
+		)
+		req.NoError(err)
+		req.NoError(operator.deleteK8sObject(akoProject))
+
+		projectDeleted := false
+		for i := 1; i < 6; i++ {
+			err = operator.getK8sObject(
+				client.ObjectKey{Name: prepareK8sName2(projectName), Namespace: operatorNamespace},
+				akoProject,
+				true,
+			)
+
+			if err != nil {
+				projectDeleted = true
+				break
+			}
+
+			time.Sleep(10 * time.Second)
+		}
+
+		if !projectDeleted {
+			t.Errorf("project %s was not cleaned up", projectName)
+		}
+
+		cleanUpKeys(t, operator, operatorNamespace, cliPath)
+	})
+
+	t.Run("should install operator importing atlas existing resources", func(t *testing.T) {
+		g := newAtlasE2ETestGenerator(t)
+		g.enableBackup = true
+		g.generateProject("k8sOperatorInstall")
+		g.generateCluster()
+
+		clusterName := "install-import"
+		operator := setupCluster(t, clusterName, operatorNamespace)
+		context := fmt.Sprintf("kind-%s", clusterName)
+
+		cmd := exec.Command(cliPath,
+			"kubernetes",
+			"operator",
+			"install",
+			"--targetNamespace", operatorNamespace,
+			"--projectName", g.projectName,
+			"--import",
+			"--kubeContext", context)
+		cmd.Env = os.Environ()
+		resp, inErr := cmd.CombinedOutput()
+		req.NoError(inErr, string(resp))
+		a.Equal("Atlas Kubernetes Operator installed successfully\n", string(resp))
+
+		checkDeployment(t, operator, operatorNamespace)
+
+		akoProject := akov1.AtlasProject{}
+		err = operator.getK8sObject(
+			client.ObjectKey{Name: prepareK8sName2(g.projectName), Namespace: operatorNamespace},
+			&akoProject,
+			true,
+		)
+		req.NoError(err)
+
+		akoDeployment := akov1.AtlasDeployment{}
+		err = operator.getK8sObject(
+			client.ObjectKey{Name: prepareK8sName2(fmt.Sprintf("%s-%s", g.projectName, g.clusterName)), Namespace: operatorNamespace},
+			&akoDeployment,
+			true,
+		)
+		req.NoError(err)
+
```

Comment:
It would be nice to also check the status of the project and/or deployment, to see if the installer installed the operator with the correct credentials secret.